docs/public_testnet_deployment_plan.md

```diff
@@ -389,12 +385,12 @@ add eosio.code to active permission
 
 transfer initial balance
 ```
-./cleos transfer eosio evmevmevmevm "1.0000 EOS" "evmevmevmevm"
+./cleos transfer testaccount evmevmevmevm "1.0000 A" "evmevmevmevm"
```

Comment:
Make sure this actually works. I don't remember if the current version of cleos intelligently picks the correct contract if it detects you are sending the `A` token.